e2e/cli-e2e/tests/__snapshots__/help.e2e.test.ts.snap

```diff
@@ -11,26 +11,88 @@ Commands:
   code-pushup print-config  Print config
 
+Global Options:
+      --progress  Show progress bar in stdout.         [boolean] [default: true]
+      --verbose   When true creates more verbose output. This is helpful when de
+                  bugging.                            [boolean] [default: false]
+      --config    Path to config file, e.g. code-pushup.config.ts. By default it
+                   loads code-pushup.config.(ts|mjs|js).                [string]
+      --tsconfig  Path to a TypeScript config, to be used when loading config fi
+                  le.                                                   [string]
+
+Persist Options:
+      --persist.outputDir  Directory for the produced reports           [string]
+      --persist.filename   Filename for the produced reports.           [string]
+      --persist.format     Format of the report output. e.g. \`md\`, \`json\`[array]
+
+Upload Options:
+      --upload.organization  Organization slug from portal              [string]
+      --upload.project       Project slug from portal                   [string]
+      --upload.server        URL to your portal server                  [string]
+      --upload.apiKey        API key for the portal server              [string]
+
 Options:
-      --progress             Show progress bar in stdout.
-                                                       [boolean] [default: true]
-      --verbose              When true creates more verbose output. This is help
-                             ful when debugging.      [boolean] [default: false]
-      --config               Path to config file, e.g. code-pushup.config.ts. By
-                              default it loads code-pushup.config.(ts|mjs|js).
-                                                                        [string]
-      --tsconfig             Path to a TypeScript config, to be used when loadin
-                             g config file.                             [string]
-      --persist.outputDir    Directory for the produced reports         [string]
-      --persist.filename     Filename for the produced reports.         [string]
-      --persist.format       Format of the report output. e.g. \`md\`, \`json\`
-                                                                         [array]
+      --onlyPlugins  List of plugins to run. If not set all plugins are run.
+                                                           [array] [default: []]
+  -h, --help         Show help                                         [boolean]
+
+Examples:
+  code-pushup collect --config tsconfig.te  Use the config from \`tsconfig.test.j
+  st.json                                   son\` and only collect audits
+  code-pushup print-config --onlyPlugins    print the config object but from the
+                                             plugins only list given slugs
+"
+`;
+
+exports[`CLI help > should print help with command help 1`] = `
```

Comment:
The `CLI help > should print help with argument ---help 1` snapshot above is now obsolete, so should be deleted.

Reply:
added test instead of deleting